packages/js/block-templates/src/hooks/use-layout-template/use-layout-template.ts

```diff
@@ -49,11 +49,11 @@ export const useLayoutTemplate = ( layoutTemplateId: string | null ) => {
 		// To prevent this, we pass `__invalid-template-id` as the ID when there is no layout template ID.
 		// A request will still be triggered, but it will return no results.
 		layoutTemplateId || '__invalid-template-id',
-		// Only perform the query if we have a layout template ID; otherwise, just return null.
+		// Only perform the query if the layout template entity is registered and we have a layout template ID; otherwise, just return null.
 		// Note: Until we are using @woocommerce/core-data 6.24.0 (Gutenberg 17.2),
 		// the REST API requests will still be triggered even when the query is disabled due to a regression.
 		// See: https://github.com/WordPress/gutenberg/pull/56108
-		{ enabled: isEntityRegistered }
+		{ enabled: isEntityRegistered && !! layoutTemplateId }
```

Comment:
We did have the `!! layoutTemplateId` condition before, which got removed in this PR: https://github.com/woocommerce/woocommerce/pull/47678/files
Adding it back in is fine, but just want to make sure we test the scenario of that PR so that we aren't re-introducing errors. ( This seemed to be related to the latest version of GB, so testing with and without GB would be helpful )
I think

Reply:
Agreed. Looking at the PR where it was removed and the code, it seems clear to me that it was an error to remove that. I'll double-check though. Thanks for making sure we are careful here.

Reply:
I verified that the scenario in #47678 still works, both with GB and without.